### PR TITLE
Add pagination to recent order view

### LINKED ISSUE #1432: Missing pagination in "Recent orders" view
Results in the "Recent Orders" tab in "Your account" view are not paginated.

### PR DESCRIPTION
**Pull request by akjanik**
I want to merge this change because...

Fixes #1432 

screenshot 2

screenshot 1

### Pull Request Checklist

(Please keep this section. It will make maintainer's life easier.)

1. [ ] Privileged views and APIs are guarded by proper permission checks.
1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Database queries are optimized and the number of queries is constant.
1. [ ] The changes are tested.
1. [ ] The code is documented (docstrings, project documentation).
1. [ ] Python code quality checks pass: `pycodestyle`, `pydocstyle`, `pylint`.
1. [ ] JavaScript code quality checks pass: `eslint`.

### CODE REVIEW

File: saleor/userprofile/views.py [hunk 1]
```diff
@@ -8,15 +8,17 @@
...
+from ..core.utils import get_paginator_items
 from .forms import (
     ChangePasswordForm, get_address_form, logout_on_password_change)
 
-
 @login_required
 def details(request):
     password_form = get_or_process_password_form(request)
+    orders  = request.user.orders.prefetch_related('groups__lines')
+    orders_paginated = get_paginator_items(orders, 5, request.GET.get('page'))
```
Comment (patrys) [inline saleor/userprofile/views.py]: Why 5?

Comment (elwoodxblues) [reply saleor/userprofile/views.py]: @akjanik There is PAGINATE_BY setting in settings.py, which is used across the storefront.

Comment (akjanik) [reply saleor/userprofile/views.py]: I was about to ask in this PR. Didn't know what number to pick up, `settings.PAGINATE_BY` is 16, isn't too large?

Comment (akjanik) [reply saleor/userprofile/views.py]: ^ok then

**Comment by elwoodxblues** [comment]
Pagination links are too close to the orders table, there should be more spacing between them. Also,  changing tab to "Change password" and then back to "Recent orders" results with this bug:
image

**Comment by elwoodxblues** [comment]
@dominik-zeglen Take a look when you have time.

**Comment by elwoodxblues** [comment]
According to @dominik-zeglen, the issue with switching tabs should we resolved after upgrading Bootstrap to the latest version: #1498.

**Pull request merged**